validator/slashing-protection/local/standard-protection-format/import.go

```diff
@@ -0,0 +1,127 @@
+package interchangeformat
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+
+	"github.com/pkg/errors"
+	"github.com/prysmaticlabs/prysm/validator/db"
+	"github.com/prysmaticlabs/prysm/validator/db/kv"
+)
+
+// ImportStandardProtectionJSON takes in EIP-3076 compliant JSON file used for slashing protection
+// by eth2 validators and imports its data into Prysm's internal representation of slashing
+// protection in the validator client's database. For more information, see the EIP document here:
+// https://eips.ethereum.org/EIPS/eip-3076.
+func ImportStandardProtectionJSON(ctx context.Context, validatorDB db.Database, r io.Reader) error {
+	encodedJSON, err := ioutil.ReadAll(r)
+	if err != nil {
+		return errors.Wrap(err, "could not read slashing protection JSON file")
+	}
+	interchangeJSON := &EIPSlashingProtectionFormat{}
+	if err := json.Unmarshal(encodedJSON, interchangeJSON); err != nil {
+		return errors.Wrap(err, "could not unmarshal slashing protection JSON file")
+	}
+
```

Comment:
When importing the interchange format, you need to also check if the version is one which prysm supports. Ex: If its 3, we should return an error rather than continue processing it.